spec/controllers/owned_tag_sets_controller_spec.rb

```diff
@@ -104,6 +104,13 @@
   end
 
   describe "show" do
+    context "where tag set is not found" do
+      it "redirects and displays a notice" do
+        get :show, id: 12345
```

Comment:
I think it's fine as is -- the _ is more helpful if you're dealing with a number like 1000000000 and your need it to be easily read by humans. It doesn't serve much purpose when making up a random id for a test.